zppy/templates/coupled_global.py

```diff
@@ -184,11 +238,11 @@ def plot_global_surface_air_temperature(ax, xlim, exps, rgn):
         "lw": 1.0,
         "ohc": False,
         "set_axhline": False,
-        "set_legend": False,
+        "set_legend": True,
```

Comment:
Some plots weren't showing the legend, so I set them to show it, for better readability.